### ISSUE TITLE
Connect a Phone directly to the ESP32

### ISSUE THREAD

**Issue by jo-nas** [espressif/esp-mdf#7]
Is it possible to connect a phone via bluetooth or Wifi directly to the mesh without a Router?

**Comment by ESP-LBB** [comment]
Yes it's feasible, but we don't have the demo code for now. You can try it by yourself or wait for some time.
Thanks.

**Comment by jo-nas** [comment]
Thank you for your fast answer.

Is it maybe possible to connect to the esp32 via bluetooth? That would be a perfect solution

Mesh via WiFi and the phone is connected via bluetooth
